### PR TITLE
XCONY - added missing class and fixed smarty-variables

### PR DESCRIPTION
**Pull request by xcony**
-missing class "SloganPrinter" added in the docs
-wrong smarty variablenames fixed

Signed-off-by: cedrix <kassierer@cedrix.com>

### CODE REVIEW

File: source/developers-guide/plugin-quick-start/index.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ class FrontendSubscriber implements SubscriberInterface
         ];
     }
 
-    public function myDetailCallback(\Enlight_Controller_ActionEventArgs $args)
+    public function myDetailCallback(Enlight_Controller_ActionEventArgs $args)
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: I think this is necessary as the Subscriber should have the namespace `namespace SwagSloganOfTheDay\Subscriber`
Like defined in the services.xml

File: source/developers-guide/plugin-quick-start/index.md [hunk 2]
```diff
@@ -224,7 +247,7 @@ For this we add a folder Subscriber in which we create a new PHP class `RouteSub
 namespace SwagSloganOfTheDay\Subscriber;
 
 use Enlight\Event\SubscriberInterface;
-use Shopware\Components\Plugin\ConfigReader;
+use Enlight_Controller_Action;use Enlight_Controller_ActionEventArgs;use Shopware\Components\Plugin\ConfigReader;
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: missing line breaks

File: source/developers-guide/plugin-quick-start/index.md [hunk 3]
```diff
@@ -248,9 +271,9 @@ class RouteSubscriber implements SubscriberInterface
        $this->config = $configReader->getByPluginName($pluginName);
    }
 
-   public function onPostDispatch(\Enlight_Controller_ActionEventArgs $args)
+   public function onPostDispatch(Enlight_Controller_ActionEventArgs $args)
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: see comment above

File: source/developers-guide/plugin-quick-start/index.md [hunk 4]
```diff
@@ -606,7 +629,7 @@ SwagSloganOfTheDay
 <?php
 namespace SwagSloganOfTheDay;
 
-class SwagSloganOfTheDay extends \Shopware\Components\Plugin
+use Enlight_Controller_EventArgs;use Shopware\Components\Plugin;class SwagSloganOfTheDay extends Plugin
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: missing linebreaks

File: source/developers-guide/plugin-quick-start/index.md [hunk 5]
```diff
@@ -248,9 +271,9 @@ class RouteSubscriber implements SubscriberInterface
        $this->config = $configReader->getByPluginName($pluginName);
    }
 
-   public function onPostDispatch(\Enlight_Controller_ActionEventArgs $args)
+   public function onPostDispatch(Enlight_Controller_ActionEventArgs $args)
    {
-       /** @var \Enlight_Controller_Action $controller */
+       /** @var Enlight_Controller_Action $controller */
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: see comment above

File: source/developers-guide/plugin-quick-start/index.md [hunk 6]
```diff
@@ -704,9 +727,9 @@ By overwriting the `build()`-method the `ContainerBuilder` can be extended:
 <?php
 namespace SwagSloganOfTheDay;
 
-use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Shopware\Components\Plugin;use Symfony\Component\DependencyInjection\ContainerBuilder;
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: missing line breaks

File: source/developers-guide/plugin-quick-start/index.md [hunk 7]
```diff
@@ -615,7 +638,7 @@ class SwagSloganOfTheDay extends \Shopware\Components\Plugin
         ];
     }
 
-    public function onRouteStartup(\Enlight_Controller_EventArgs $args)
+    public function onRouteStartup(Enlight_Controller_EventArgs $args)
```
Comment (mitelg) [inline source/developers-guide/plugin-quick-start/index.md]: see comment above

**Comment by xcony** [comment]
@mitelg The "use" staff was committed by mistake. Thanks for your hint.

**Comment by mitelg** [comment]
thanks for your contribution @xcony :+1: :tada: :blue_heart:

**Pull request merged**